src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -0,0 +1,24 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+       xmlns:aop="http://www.springframework.org/schema/aop" xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+                           http://www.springframework.org/schema/tx
+                           http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
+                           http://www.springframework.org/schema/aop
+                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+    <bean id="cur-ar" parent="arModuleConfiguration-parentBean">
+        <property name="dataDictionaryPackages">
+            <list merge="true">
+                <value>classpath:edu/cornell/kfs/module/ar/businessobject/datadictionary/*.xml</value>
+            </list>
+        </property>
+        <property name="databaseRepositoryFilePaths">
+            <list merge="true">
+                <value>edu/cornell/kfs/module/ar/cu-ojb-ar.xml</value>
+            </list>
+        </property>
+    </bean>
+
```

Comment:
Also need to add:
```
	<bean id="cu-arModuleService" parent="arModuleService-parentBean">
	 	<property name="moduleConfiguration" ref="cu-ar"/>
	</bean>
```
so that the arModuleService point to the new ar module configuration.